test/getWafRulesByTags.test.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const nock = require('nock');
+const expect = require('expect');
+const config = require('../src/config');
+const fastlyPromises = require('../src/index');
+const response = require('./response/getWafRulesByTags.response');
+
+describe('#getWafRulesByTags', () => {
+  const fastly = fastlyPromises('923b6bd5266a7f932e41962755bd4254', '79CEhEeP8DKPQQGTXokV5M');
+  let res;
+
+  nock(config.mainEntryPoint)
+    //.log(console.log)
```

Comment:
Shouldn't we delete this log comment?

Reply:
I thought its easier to have it here because I _will_ forget it the next time I wanna debug stuff.